docs/Android.md

```diff
@@ -74,16 +74,18 @@ Android NDK, as well as the directories that contain the `libicuucswift.so` and
 
 ```
 $ ARM_DIR=path/to/libicu-libiconv-android
-$ NDK_PATH=path/to/android-ndk16
+$ NDK_PATH=path/to/android-ndk21
 $ utils/build-script \
     -R \                                       # Build in ReleaseAssert mode.
     --android \                                # Build for Android.
-    --android-ndk $NDK_PATH \   # Path to an Android NDK.
+    --android-ndk $NDK_PATH \                  # Path to an Android NDK.
+    --android-arch armv7 \                     # Optionally specify Android architecture, alternately aarch64
     --android-api-level 21 \                   # The Android API level to target. Swift only supports 21 or greater.
     --android-icu-uc ${ARM_DIR}/libicuucswift.so \
     --android-icu-uc-include ${ARM_DIR}/icu/source/common \
     --android-icu-i18n ${ARM_DIR}/libicui18nswift.so \
-    --android-icu-i18n-include ${ARM_DIR}/icu/source/i18n
+    --android-icu-i18n-include ${ARM_DIR}/icu/source/i18n \
+    --android-icu-data ${ARM_DIR}/libicudataswift.so
```

Comment:
This was made a requirement in #20082, but these commands weren't updated to show that, so the current commands in this doc don't work.